scania-digital-design-system/sdds-website
# Added new scania package name

- update scania package organisation
- Update all imported packages
- Update README file


**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: https://github.com/scania-digital-design-system/sdds/issues/37
https://github.com/scania-digital-design-system/sdds/pull/52
**How to test**  
- Run project and see if there's any error
- Check README information, and make sure all package name has been renamed
